terra_notebook_utils/drs.py

```diff
@@ -224,6 +224,14 @@ def _resolve_bucket_target(url: str, info: DRSInfo) -> Tuple[str, str]:
         key = pfx
     return bucket_name, key
 
+
+def _resolve_azure_blob_path(azure_url: str) -> Tuple[str, str, str]:
+    assert azure_url.startswith("https://")
+    storage_account_with_endpoint, container_and_blob_name = azure_url[8:].split("/", 1)
+    storage_account = storage_account_with_endpoint.split(".", 1)[0]
+    container, blob_name = container_and_blob_name.split("/", 1)
```

Comment:
Suggestion: could this be wrapped in a `try`/`except` so that in case an error occurs, it will raise an error including the blob path it was attempting to parse for debugging?

Something like:

```
def _resolve_azure_blob_path(azure_url: str) -> Tuple[str, str, str]:
    try:
        assert azure_url.startswith("https://")
        storage_account_with_endpoint, container_and_blob_name = azure_url[8:].split("/", 1)
        storage_account = storage_account_with_endpoint.split(".", 1)[0]
        container, blob_name = container_and_blob_name.split("/", 1)
    except:
        logger.warning(f'Error parsing azure blob at: {azure_url}')
        raise
```